src/components/ProgramRecordsList.jsx

```diff
@@ -0,0 +1,123 @@
+      >
+        <ChevronLeft className="mb-1" />
+        <FormattedMessage
+          id="records.profile.link"
+          defaultMessage="Back to My Profile"
+          description="Link text that redirects logged-in user to their profile page"
+        />
+      </Hyperlink>
+    );
+  };
+
+  const renderCredentialsServiceIssueAlert = () => (
+    <div tabIndex="-1">
+      <Alert variant="danger">
+        <Info className="text-danger-500 mr-2 mb-1" />
+        <FormattedMessage
+          id="records.list.error"
+          defaultMessage="An error occurred attempting to retrieve your program records. Please try again later."
+          description="An error message indicating there is a problem retrieving the user's program records"
+        />
+      </Alert>
+    </div>
+  );
+
+  const renderEmpty = () => (
+    <p className="pl-3 pr-3">
+      <FormattedMessage
+        id="records.list.empty"
+        defaultMessage="No records yet. Program records are created once you have earned at least one course
+        certificate in a program."
+        description="A message indicating the user has no program records to display on the Learner Records page"
+      />
+    </p>
+  );
+
+  const [records, setRecords] = React.useState([]);
+
+  const renderRows = () => (
+    <section id="program-records-list" className="pl-3 pr-3">
+      <header>
+        <h3 className="h4">
+          <FormattedMessage
+            id="records.subheader"
+            defaultMessage="Program Records"
+            description="Subheader for the Learner Records page"
+          />
+        </h3>
+        <p>
+          <FormattedMessage
+            id="records.description"
+            defaultMessage="A program record is created once you have earned at least one course certificate
+            in a program."
+            description="Description of program records for the Learner Records page"
+          />
+        </p>
+      </header>
+    </section>
+  );
+
+  const renderList = () => {
+    const [isLoaded, setIsLoaded] = React.useState(false);
+    const [hasNoData, setHasNoData] = React.useState(false);
+
+    React.useEffect(() => {
+      getProgramRecords().then((data) => {
+        setIsLoaded(true);
+        if (_.isEmpty(data)) {
+          setHasNoData(true);
+        } else {
+          setRecords(data.enrolled_programs);
+        }
+      }).catch((error) => {
+        const errorMessage = (`Error: Could not fetch learner record data for user: ${error.message}`);
+        logError(errorMessage);
+      });
+    }, []);
+
+    if (isLoaded) {
+      if (hasNoData) {
+        return (renderCredentialsServiceIssueAlert());
+      }
+      if (!records.length) {
+        return (renderEmpty());
+      }
+      return (renderRows());
+    }
+    return null;
+  };
+
+  return (
+    <main id="main-content" className="pt-5 pb-5 pl-4 pr-4 " tabIndex="-1">
+      {renderProfile()}
+      <h2 className="h3 pl-3 pr-3 mb-4">
```

Comment:
I feel like starting off with an H2 here is a violation of accessibility (a11y) principles. Header levels should follow a natural progression, starting with h1. I believe the `classname` "h3" means "style this like an h3".

I think we should change this to an h1 and keep the h3 styling here. @Tj-Tracy @oliviaruizknott do I have this right?

Reply:
Oooh, I think you’re right, @justinhynes: we’re supposed to start with h1s and work our way down, semantically. (You can do whatever you want with styling.) I also know that there’s only supposed to be one h1 on a page. As to exactly how everything should be tagged here, I’m not sure! Try running this by @wittjeff (@jwitt on Slack, or post in the #accessibility channel).